### ISSUE TITLE
Distinguish dataset types from PwC

### ISSUE THREAD

**Issue by mbollmann** [acl-org/acl-anthology#1790]
> I realise I have no power here, but it seems unintuitive for all datasets a paper uses & introduces to be listed on the anthology without distinction. Would a richer response from PwC, that distinguishes between the types, be better?

_Originally posted by @leondz in https://github.com/acl-org/acl-anthology/issues/1789#issuecomment-1020877498_

Context here is the observation that Papers with Code apparently [can make a distinction between datasets "Introduced in" vs. "Used in" a paper](https://paperswithcode.com/paper/offensive-language-and-hate-speech-detection), but we don't show this distinction on the Anthology [nor does the PwC endpoint include it](https://github.com/acl-org/acl-anthology/pull/1789#issuecomment-1020591824).

I'm taking the liberty of turning this into its own issue so that more people see it, and I can tag @rstojnic to ask whether this would even be possible/easy to add :)

**Comment by rstojnic** [comment]
Yes we could make that this distinction on ACL as well, we haven't initially just to keep that section more simple and compact - but I can see how it can be confusing. On the arXiv Code&Data tab (https://arxiv.org/abs/1908.04531v1) it looks like this:

![image](https://user-images.githubusercontent.com/7289878/150962857-46549ca3-c2e9-4bfa-a885-acb5b6ecce46.png)

So if the current layout on ACL is confusing we could perhaps show it as 
```
Data: DKhate (newly introduced), OLID
```

What do you think?

**Comment by rstojnic** [comment]
Also tagging in @leondz who originally reported this.

**Comment by leondz** [comment]
The PwC presentation is a little more verbose but a lot less ambiguous than the arXiv one:
<img width="256" alt="Screenshot 2022-01-25 at 13 56 13" src="https://user-images.githubusercontent.com/121934/150980864-6994aa2f-f8e6-42fd-b1fa-41c2176e2407.png">

For example, the scoping of "used" in the arXiv example seems misleadingly broad

**Comment by mbollmann** [comment]
No longer relevant after the sunsetting of PwC.
